src/jdk.compiler/share/classes/com/sun/tools/javac/code/Lint.java

```diff
@@ -410,10 +414,14 @@ public enum LintCategory {
             this(option, true);
         }
 
-        LintCategory(String option, boolean annotationSuppression) {
+        LintCategory(String option, boolean annotationSuppression, String... aliases) {
             this.option = option;
             this.annotationSuppression = annotationSuppression;
-            map.put(option, this);
+            ArrayList<String> optionList = new ArrayList<>(1 + aliases.length);
+            optionList.add(option);
+            Stream.of(aliases).forEach(optionList::add);
```

Comment:
```suggestion
            Collections.addAll(optionList, aliases);
```

Reply:
Wow, somehow I never knew `Collections.addAll()` existed. Thanks for the tip - that's simpler.

(I'll let @vicente-romero-oracle fix it.)